ls1intum/Artemis
# Enable compass for exam exercises

### Checklist

### Motivation and Context
Compass was disabled for exam's but since we changed the workflow of Compass it can now be enabled with this thesis.

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Administration
3. Create an exam
4. Create exercise group and create a modeling exercise which automatic assessment suggestions enabled
5. Create student exams
6. Change the working times of the students
7. Submit similar solutions in the students exams
8. Assess after the latest of the startTime + working time + gracePeriod (180 seconds default) + 1 minute
9. Automatic assessments should appear after the first submission
10. You can check number of clusters in modeling exercise detail page with an admin account, number should be more than 0
